README.md

```diff
@@ -53,32 +53,49 @@ ln.finish()  # finish the run
 
 <!-- from here on, slight deviation from preface.md, where all this is treated in the walk through in more depth -->
 
-Running this code inside a script via `python create-fasta.py` produces the following data lineage.
+This code snippet creates an artifact, which can store a dataset or model as a file or folder in various formats.
+Running the snippet as a script (`python create-fasta.py`) produces the following data lineage.
 
 ```python
 artifact = ln.Artifact.get(key="sample.fasta")  # query artifact by key
 artifact.view_lineage()
 ```
 
-<img src="https://lamin-site-assets.s3.amazonaws.com/.lamindb/EkQATsQL5wqC95Wj0003.png" width="220">
+<img src="https://lamin-site-assets.s3.amazonaws.com/.lamindb/EkQATsQL5wqC95Wj0005.png" width="300">
 
-You'll know how that artifact was created.
+You'll know how that artifact was created and what it's used for ([interactive visualization](https://lamin.ai/laminlabs/lamindata/artifact/8incOOgjn6F0K1TS)) in addition to capturing basic metadata:
 
 ```python
 artifact.describe()
 ```
 
-<img src="https://lamin-site-assets.s3.amazonaws.com/.lamindb/BOTCBgHDAvwglN3U0001.png" width="850">
+<img src="https://lamin-site-assets.s3.amazonaws.com/.lamindb/BOTCBgHDAvwglN3U0002.png" width="550">
 
-Conversely, you can query artifacts by the script that created them.
+You can also query artifacts by the script that created them.
 
 ```python
-ln.Artifact.get(transform__key="create-fasta.py")  # query artifact by transform key
+ln.Artifact.filter(transform__key="create-fasta.py").to_dataframe()  # query artifacts by transform key
 ```
 
 Data lineage is just one type of metadata to help analysis and model training through queries, validation, and annotation. Here is a more [comprehensive example](https://lamin.ai/laminlabs/lamindata/artifact/9K1dteZ6Qx0EXK8g).
 
-<img src="https://lamin-site-assets.s3.amazonaws.com/.lamindb/6sofuDVvTANB0f480001.png" width="850">
+<img src="https://lamin-site-assets.s3.amazonaws.com/.lamindb/6sofuDVvTANB0f480002.png" width="850">
+
+If you just want to annotate an artifact with a label, use the universal label ontology:
```

Comment:
But why would you do it? May comment at least explained why.

Reply:
We could use the "organize" word, e.g.:

"Data lineage is just one type of metadata to organize data for analysis and model training through queries, validation, and annotation."

Reply:
How about: "Beyond data lineage, you can organize datasets through validation, annotation, queries & search for any metadata."

Now it starts with "beyond". I think the "find" is implicit in "query & search". WDYT?

Reply:
> We need to stay concise and can't repeat ourselves.

I don't agree, we want to motivate the product and arose some interest in the reader. If that requires not being concise or repeating, so be it, i think.

Reply:
> we want to motivate the product and arose some interest in the reader.

Agreed.

> If that requires not being concise or repeating

I think that's a fallacy. Being motivating and triggering instance never requires not being concise or repetitive. By contrast, readers are typically annoyed if writers are not being concise and/or repetitive. Also we received such feedback already.

---

All good writers I know put in tremendous effort to arrive at very concise explanations. They use repetition with much care, in a very targeted way; and typically very rarely.

---

Our README is short enough that it should be understandable without repetition; in particular if we're being concise and hence more easily fit into the readers context window; be they human or AI.

Reply:
i mostly agree, i am just afraid of using potential users on readme if we don't explain enough.